bbq/vm/vm.go

```diff
@@ -917,8 +917,12 @@ func opInvokeMethodDynamic(vm *VM, ins opcode.InstructionInvokeDynamic) {
 	// Load type arguments
 	typeArguments := loadTypeArguments(vm, ins.TypeArgs)
 
-	// Load arguments
-	arguments := vm.popN(int(ins.ArgCount))
+	// Load arguments.
+	// Make a copy, since the slice can get mutated, since the stack is reused.
+	// This can happen in case getting the member below triggers execution of other functions.
```

Comment:
Good catch! I wonder how a member-access/ a nested function call could change the existing levels of the stack though 🤔, because the nested function calls should only use the stack from current index/position onwards. The bug / problem could actually be there, rather than here.